diff --git a/src/exports/vite/index.js b/src/exports/vite/index.js
index c2a339693909e96d47431a09ab9aa67f7616acf4..3b6214595b2e2ae7e18aba1083f26ae6d4d57393 100644
--- a/src/exports/vite/index.js
+++ b/src/exports/vite/index.js
@@ -556,9 +556,9 @@ function kit({ svelte_config }) {
 							input,
 							output: {
 								format: 'esm',
-								entryFileNames: ssr ? '[name].js' : `${prefix}/[name].[hash].${ext}`,
-								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[name].[hash].${ext}`,
-								assetFileNames: `${prefix}/assets/[name].[hash][extname]`,
+								entryFileNames: ssr ? '[name].js' : `${prefix}/[name].${ext}`,
+								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[name].${ext}`,
+								assetFileNames: `${prefix}/assets/[name][extname]`,
 								hoistTransitiveImports: false
 							},
 							preserveEntrySignatures: 'strict'